NPE handling SuggestionNameListUpdated event

[The culprit.](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/posts/ViewPostFragment.java#L83)
```
de.greenrobot.event.EventBusException: Invoking subscriber failed
       at de.greenrobot.event.EventBus.handleSubscriberException(EventBus.java:518)
       at de.greenrobot.event.EventBus.invokeSubscriber(EventBus.java:500)
       at de.greenrobot.event.EventBus.invokeSubscriber(EventBus.java:492)
       at de.greenrobot.event.HandlerPoster.handleMessage(HandlerPoster.java:67)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:212)
       at android.app.ActivityThread.main(ActivityThread.java:5151)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:868)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:684)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by: java.lang.NullPointerException
       at org.wordpress.android.ui.posts.ViewPostFragment.onEventMainThread(ViewPostFragment.java:83)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at de.greenrobot.event.EventBus.invokeSubscriber(EventBus.java:498)
       at de.greenrobot.event.EventBus.invokeSubscriber(EventBus.java:492)
       at de.greenrobot.event.HandlerPoster.handleMessage(HandlerPoster.java:67)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:212)
```

--- a/WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java
@@ -380,7 +380,7 @@ public class CommentDetailFragment extends Fragment implements NotificationFragm
     @SuppressWarnings("unused")
     public void onEventMainThread(SuggestionEvents.SuggestionNameListUpdated event) {
         // check if the updated suggestions are for the current blog and update the suggestions
-        if (event.mRemoteBlogId != 0 && event.mRemoteBlogId == mRemoteBlogId) {
+        if (event.mRemoteBlogId != 0 && event.mRemoteBlogId == mRemoteBlogId && mSuggestionAdapter != null) {
             List<Suggestion> suggestions = SuggestionTable.getSuggestionsForSite(event.mRemoteBlogId);
             mSuggestionAdapter.setSuggestionList(suggestions);
         }

--- a/WordPress/src/main/java/org/wordpress/android/ui/posts/ViewPostFragment.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/posts/ViewPostFragment.java
@@ -78,7 +78,7 @@ public class ViewPostFragment extends Fragment {
     public void onEventMainThread(SuggestionEvents.SuggestionNameListUpdated event) {
         int remoteBlogId = WordPress.getCurrentRemoteBlogId();
         // check if the updated suggestions are for the current blog and update the suggestions
-        if (event.mRemoteBlogId != 0 && event.mRemoteBlogId == remoteBlogId) {
+        if (event.mRemoteBlogId != 0 && event.mRemoteBlogId == remoteBlogId && mSuggestionAdapter != null) {
             List<Suggestion> suggestions = SuggestionTable.getSuggestionsForSite(event.mRemoteBlogId);
             mSuggestionAdapter.setSuggestionList(suggestions);
         }

--- a/WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderCommentListActivity.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderCommentListActivity.java
@@ -144,7 +144,7 @@ public class ReaderCommentListActivity extends ActionBarActivity {
     @SuppressWarnings("unused")
     public void onEventMainThread(SuggestionEvents.SuggestionNameListUpdated event) {
         // check if the updated suggestions are for the current blog and update the suggestions
-        if (event.mRemoteBlogId != 0 && event.mRemoteBlogId == mBlogId) {
+        if (event.mRemoteBlogId != 0 && event.mRemoteBlogId == mBlogId && mSuggestionAdapter != null) {
             List<Suggestion> suggestions = SuggestionTable.getSuggestionsForSite(event.mRemoteBlogId);
             mSuggestionAdapter.setSuggestionList(suggestions);
         }

